src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/EventLoggingService.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.personrecord.service
+
+import com.microsoft.applicationinsights.TelemetryClient
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.EventLoggingEntity
+import uk.gov.justice.digital.hmpps.personrecord.jpa.repository.EventLoggingRepository
+import java.time.LocalDateTime
+
+@Service
+class EventLoggingService(
+  private val eventLoggingRepository: EventLoggingRepository,
+  private val telemetryClient: TelemetryClient,
+
+) {
+
+  fun mapToEventLogging(
+    beforeData: String? = null,
+    processedData: String? = null,
+    sourceSystemId: String? = null,
+    uuid: String? = null,
+    sourceSystem: String? = null,
+    eventType: String? = null,
+    eventTimeStamp: LocalDateTime? = null,
```

Comment:
I think this is always `localDateTime.now()` - let's just use that instead of passing it in